pkg/healthcheck/healthcheck.go

```diff
@@ -38,13 +38,13 @@ type HealthChecker interface {
 	HealthCheck(ctx context.Context, cs *acsapi.OpenShiftManagedCluster) error
 }
 
-type simpleHealthChecker struct{}
+type simpleHealthChecker struct{ pluginConfig *acsapi.PluginConfig }
```

Comment:
If you only need to use `AcceptLanguages`, pass just that instead of the whole PluginConfig.